docs/config.json

```
@@ -81,7 +81,7 @@
               "slug": "/setup/admin/graceful-restarts/"
             },
             {
-              "title": "Daemon",
+              "title": "Run Teleport as a Daemon",
```

Nit: Not sure daemon should be capitalized.
```suggestion
              "title": "Run Teleport as a daemon",
```